### PR TITLE
Moving connector tests to multiple splits.

### PR DESCRIPTION
**Pull request by naman223**
Fixing #22102 

Splitting them to multiple tests

### CODE REVIEW

**Comment by naman223** [comment]
@glassfishrobot Run CI tests please

File: appserver/tests/appserv-tests/devtests/connector/v3/run_test.sh [hunk 1]
```diff
@@ -51,9 +51,9 @@ test_run(){
   echo $ROOT
   cd $APS_HOME/../v2-tests/appserv-tests
```
Comment (arindam-bandyopadhyay) [inline appserver/tests/appserv-tests/devtests/connector/v3/run_test.sh]: /v2-tests/appserv-tests folder would be deleted soon. That is the whole purpose of moving test from v2-tests/appserv-tests to appserv-tests . Please remove this reference

File: appserver/tests/appserv-tests/devtests/connector/v3/run_test.sh [hunk 2]
```diff
@@ -51,9 +51,9 @@ test_run(){
...
   ant startDomain startDerby
-  antTarget="clean-all clean all report"
   cd $ROOT
-  time ant $antTarget | tee $TEST_RUN_LOG
+  echo Running target: $TARGET
+  time ant clean-all clean $TARGET report | tee $TEST_RUN_LOG
   antStatus=$?
   cp connector.output tests-run.log
   cd $APS_HOME/../v2-tests/appserv-tests
```
Comment (arindam-bandyopadhyay) [inline appserver/tests/appserv-tests/devtests/connector/v3/run_test.sh]: /v2-tests/appserv-tests folder would be deleted soon. That is the whole purpose of moving test from v2-tests/appserv-tests to appserv-tests . Please remove this reference

**Review by arindam-bandyopadhyay — CHANGES_REQUESTED** [review]

**Comment by naman223** [comment]
@glassfishrobot Run CI tests please

**Comment by arindam-bandyopadhyay** [comment]
@glassfishrobot Run CI tests please

**Review by arindam-bandyopadhyay — APPROVED** [review]

**Pull request merged**